src/main/java/io/cdap/plugin/gcp/gcs/source/GCSSource.java

```diff
@@ -290,6 +296,11 @@ public boolean isCopyHeader() {
       return copyHeader != null && copyHeader;
     }
 
+    @Override
+    public boolean skipHeader() {
+      return skipHeader == null ? false : skipHeader;
```

Comment:
this will be null in the configure method is skipHeader is a macro. Are we only calling this from prepareRun() or are we calling it in configure too? If only in prepare, we're all good.

Reply:
This isn't called at all at this moment, the value is set using reflections for format plugins